Topics/Tech_Stacks/Cypress.md

```diff
@@ -0,0 +1,65 @@
+## E2E Testing with Cypress
+
+# Cypress Introduction
+
+Cypress is mainly used for testing web applications, especially those built in javascript. It provides an interface to programatically test your application, and visually what went wrong (or right) in tests. 
+  
+# Why do end to end testing?  
+  
+[https://circleci.com/blog/what-is-end-to-end-testing/](https://circleci.com/blog/what-is-end-to-end-testing/)
+
+The above link has a good explanation on what end to end testing is and why it should be used. 
```

Comment:
Rather than just providing a link, it might be better to also provide a summary of the main points of the linked article explaining the importance of end-to-end testing. This way, the reader could get a quick understanding without needing to navigate away, and if they wanted to learn more, they could then click the provided link.

Reply:
I think this is a great idea, added that